p4src/tna/include/control/upf.p4

```diff
@@ -172,6 +171,8 @@ control UpfIngress(
         ue_session_id = fabric_md.routing_ipv4_dst;
         session_meter_idx_internal = session_meter_idx;
         fabric_md.bridged.upf.tun_peer_id = tun_peer_id;
+        // When sending to dbuf, even if we don't drop traffic, we want to skip
+        // the egress UPF counter.
```

Comment:
This is somewhat obvious from the code. It would be more helpful to say why we skip the egress counter. Perhaps something like "Sending to dbuf means buffering, while the egress counter is used only to keep track of packets effectively sent to UEs. We will update it when draining from dbuf."